tests/venv/binder-dir/binder/postBuild

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-jupyter nbextension enable --py --sys-prefix ipyleaflet
+date
```

Comment:
I think for this test we need a postBuild that has an effect we can test for, so we are verifying that the script executed

Reply:
Good point, I've changed this to create a file that can be checked in verify